Algorithms/Search/JumpSearcher.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Algorithms.Searches
+{
+    public class JumpSearcher
```

Comment:
Could you make a generic version like [here](https://github.com/TheAlgorithms/C-Sharp/blob/master/Algorithms/Search/BinarySearcher.cs)? It allows to abstract from numbers and leaves only important features like element comparability